src/ui/public/agg_types/__tests__/agg_param_writer.js

```diff
@@ -1,132 +0,0 @@
-/*
```

Comment:
Are these obsolete tests? Do we need to substitute them with new tests?

Reply:
these are tests for the agg_param_writer class which has not been in use for a while.

i removed it and also its tests.

Reply:
I am confused, I think that was actually just a helper class for testing if parameter writing works properly? At least those tests imho still had a use-case. I actually added a couple of new tests in https://github.com/elastic/kibana/pull/20519 which seemed to work fine to test, if the parameter write functions do what they should do.

If we remove that version of those test, I think we should build some replacement, for checking if parameter writing will actually return in the correct results.